packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts

```diff
@@ -153,8 +153,8 @@ const getPanelBreakpoints = (type: PanelType): { [x: string]: IStyle } | undefin
 const commandBarHeight = '44px';
 
 const sharedPaddingStyles = {
-  paddingLeft: '16px',
-  paddingRight: '16px'
+  paddingLeft: '24px',
+  paddingRight: '24px'
```

Comment:
This change is the only one that is worrisome that it's not messing layouts. It's a 16px reduction in the overall width that was available to products to fit things in the panel and the probability that the entire width was taken into account by designers is high.

Reply:
@micahgodbolt partners need to know about this surface area reduction. It might easily break some layouts if the styling of the content inside was always counting on a specific width.

Reply:
Yeah, part of this change is partner communication on Max's part.